app/constants/image.js

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export const DEFAULT_MIME_TYPE = 'image/png';
+
+export const mimeTypeExtensions = {
+    'image/png': '.png',
+    'image/gif': '.gif',
+    'image/jpeg': '.jpeg',
```

Comment:
Please add other supported types like jpg, bmp, tiff

Reply:
Maybe take a look at the mimedb lib that we have as a dependency, perhaps that can be of help